### PR TITLE
Digisam 151 config

### PR DESCRIPTION
**Pull request by danege**
Use only YAML-file settings for configuration

### CODE REVIEW

**Review by tokee — CHANGES_REQUESTED** [review]
You forgot to update `ds-cumulus-export.yml` :grinning: 

...Oh, you just fixed it! Never mind.

File: src/main/java/dk/kb/ds/cumulus/export/Configuration.java [hunk 1]
```diff
@@ -115,11 +122,15 @@ public static String getCollection(){
...
     public static String getType(){
         return instance().type;
     }
-
+    public static String getLimited(){
+        return instance().limited;
+    }
+    public static String getCounter(){
+        return instance().counter;
+    }
```
Comment (tokee) [inline src/main/java/dk/kb/ds/cumulus/export/Configuration.java]: While we are cleaning up: The key names are not very descriptive and I find it cumbersome to have 2 keys controlling the amount of output. Convention has it that `-1` means "no limit", so how about switching to a single key called `maxrecords`, `recordlimit` or something like that?

File: src/main/java/dk/kb/ds/cumulus/export/Configuration.java [hunk 2]
```diff
@@ -115,11 +122,15 @@ public static String getCollection(){
...
     }
-
     public static String getType(){
         return instance().type;
     }
-
+    public static String getLimited(){
+        return instance().limited;
+    }
+    public static String getCounter(){
```
Comment (tokee) [inline src/main/java/dk/kb/ds/cumulus/export/Configuration.java]: When we know the type of a specific property (in this case that `counter`is an integer) have a first-claass support for that property (the method `getCounter`), the method should take care of parsing and return the value as the type that is expected: `public static int getCounter()`.

**Review by tokee — CHANGES_REQUESTED** [review]

File: src/main/java/dk/kb/ds/cumulus/export/Configuration.java (hunk 2 above)
Comment (danege) [reply src/main/java/dk/kb/ds/cumulus/export/Configuration.java]: Corrected in new commit

Comment (tokee) [reply src/main/java/dk/kb/ds/cumulus/export/Configuration.java]: Great! Thanks.

**Pull request merged**